Modules/_io/winconsoleio.c

```diff
@@ -263,7 +263,7 @@ _io__WindowsConsoleIO___init___impl(winconsoleio *self, PyObject *nameobj,
     int fd_is_own = 0;
     HANDLE handle = NULL;
 
-#ifdef Py_DEBUG
+#ifdef NDEBUG
```

Comment:
Hum, that's wrong :-( You should only emit assertion if the NDEBUG macro is **not** defined... Double negation, I hate that. With the current code, in short you declare the state variable which is not used, but the assertion is always skipped!

Reply:
Argh! Fix-up of the fix-up in flight.

Reply:
Let's await Victor's review this time :)